<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1135</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RobotUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="dbgTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>741</width>
      <height>181</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="conIPBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addIPEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="sendDataBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>420</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="cVideoLabel" name="videoLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QToolButton" name="playVideoBtn">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="stopVideoBtn">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>50</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1135</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actConIPAddr">
   <property name="text">
    <string>Connect to IP Address</string>
   </property>
   <property name="toolTip">
    <string>Connect to IP Address</string>
   </property>
  </action>
  <action name="actSendData">
   <property name="text">
    <string>Send Data</string>
   </property>
  </action>
  <action name="actPlayVideo">
   <property name="text">
    <string>Play Video</string>
   </property>
  </action>
  <action name="actStopVideo">
   <property name="text">
    <string>Stop Video</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>cVideoLabel</class>
   <extends>QLabel</extends>
   <header>cvideolabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
